05-script.md

```diff
@@ -440,3 +440,34 @@ Of course, this introduces another tradeoff between flexibility and complexity.
 > # Script 3
 > echo $@.dat
 > ~~~
+
+> ## Ooops my large script has a bug. How can I easily find the problem? {.challenge}
+>
+> You have written a large script with many lines, and it's not working properly.
+> How can I easily identify the problem ?
```

Comment:
I tweaked the text a bit noting your comments of 'You' and 'I'. I don't mind if the title changes. Interpreting the 'output' might not be 'easy' but at least someone can look at it, hopefully spot the error and they have something to provide them with an idea as to where they went wrong.